FBC/FinancialInstruments/InstrumentPricing.rdf

```diff
@@ -0,0 +1,478 @@
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-utl-alx;hasArgument"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fbc-fi-ip;OfferPrice"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label xml:lang="en">bid ask spread</rdfs:label>
+		<skos:definition xml:lang="en">difference between an offer (ask) price and a bid price</skos:definition>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-fbc-fi-ip;BidPrice">
+		<rdfs:subClassOf rdf:resource="&fibo-fbc-fi-ip;SecurityPrice"/>
+		<rdfs:label xml:lang="en">bid price</rdfs:label>
+		<owl:disjointWith rdf:resource="&fibo-fbc-fi-ip;OfferPrice"/>
+		<skos:definition xml:lang="en">price a prospective buyer is willing to pay</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">The term &apos;bid price&apos; is used by traders / market makers with respect to a given security, and that are prepared to buy or sell round lots at publicly quoted prices, and by specialists in certain instruments that perform similar functions on an exchange.</fibo-fnd-utl-av:explanatoryNote>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-fbc-fi-ip;ClosingPriceDeterminationMethod">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-gao-obj;Strategy"/>
+		<rdfs:label xml:lang="en">closing price determination method</rdfs:label>
+		<skos:definition xml:lang="en">strategy for calculating or otherwise determining an official closing price, typically by an exchange or trading venue</skos:definition>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-fbc-fi-ip;CollectionOfPrices">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-arr-arr;DatedStructuredCollection"/>
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-utl-alx;StatisticalPopulation"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-arr-arr;hasConstituent"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fbc-fi-ip;SecurityPrice"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label xml:lang="en">collection of prices</rdfs:label>
+		<skos:definition xml:lang="en">statistical population consisting of a series of prices for some security</skos:definition>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-fbc-fi-ip;CompositeMarket">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-arr-arr;StructuredCollection"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-arr-arr;hasConstituent"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fbc-fct-mkt;Exchange"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label xml:lang="en">composite market</rdfs:label>
+		<skos:definition xml:lang="en">group of exchanges or trading venues referenced for pricing purposes</skos:definition>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-fbc-fi-ip;HighPrice">
+		<rdfs:subClassOf rdf:resource="&fibo-fbc-fi-ip;SecurityPrice"/>
+		<rdfs:label xml:lang="en">high price</rdfs:label>
+		<skos:definition xml:lang="en">highest value for a given security over the period specified</skos:definition>
```

Comment:
Security Price has an observed date and time associated with it;  I've added a restriction for date period, since it applies to some window of time, though.  Same for Low, best bid, best offer, mid-price, price spread, rate of return, and yield.  Bid and ask (offer) prices are typically not based on trades - they are OTC oriented for the most part, so I don't know what you are looking for there - a boolean? And which prices does that apply to?